### PR TITLE
[changelog] 6.0.0-rc.1

### PR DESCRIPTION
**Pull request by olivielpeau**
### What does this PR do?

Add changelog for `6.0.0-rc.1`

### Additional Notes

Also, fix to the yaml format of one of the release notes (reno was complaining about it, probably should've been caught by our CI)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,215 @@
 Release Notes
 =============
 
+6.0.0-rc.1
+==========
+
+Prelude
+-------
+
+The execution of the main agent, trace-agent (APM), and process-agent processes is now orchestrated
```
Comment (truthbk) [inline CHANGELOG.rst]: I sometimes use the prelude, but I see no one else is doing it, so we don't want to make it look like these three changes are maybe more important than others. Feel free to kill the prelude, or not.

**Review by truthbk — DISMISSED** [review]
Just a comment regarding the prelude - feel free to ignore.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,215 @@
...
+- Fix the import proxy build conversion
+
+- Fix the agent stop command
+
+- Fixes a bug that caused the GUI to create a flare without a status file
+
+- Fix https://github.com/DataDog/datadog-agent/issues/1159 where error was not
+  explicit when the check had invalid configuration or code
+
+- Fix to evaluate whether the DCA can query resources (events, ervices, nodes, pods) before running core component or checks.
```
Comment (arbll) [inline CHANGELOG.rst]: `ervices` => `services`

File: CHANGELOG.rst (hunk 1 above)
Comment (olivielpeau) [reply CHANGELOG.rst]: I'd say the orchestration and the changes to where the trace and process agent should be configured in are pretty significant changes, I don't mind leaving them here.

But you're right, we may want to remove the prelude entirely from the individual release notes (and maybe manually write "highlights" for every release to stress on the most significant changes for users?)

**Pull request merged**